quarkus/config-api/src/main/java/org/keycloak/config/HostnameOptions.java

```diff
@@ -10,11 +10,21 @@ public class HostnameOptions {
             .description("Hostname for the Keycloak server.")
             .build();
 
+    public static final Option HOSTNAME_URL = new OptionBuilder<>("hostname-url", String.class)
+            .category(OptionCategory.HOSTNAME)
+            .description("Set the base URL for frontend URLs.")
+            .build();
+
     public static final Option HOSTNAME_ADMIN = new OptionBuilder<>("hostname-admin", String.class)
             .category(OptionCategory.HOSTNAME)
             .description("The hostname for accessing the administration console. Use this option if you are exposing the administration console using a hostname other than the value set to the 'hostname' option.")
             .build();
 
+    public static final Option HOSTNAME_ADMIN_URL = new OptionBuilder<>("hostname-admin-url", String.class)
+            .category(OptionCategory.HOSTNAME)
+            .description("Set the base URL for accessing the administration console.")
```

Comment:
not sure if we should make it more precise that this is an alternative to setting `hostname-admin` and other options in this description (as @vmuzikar mentioned yesterday). 

e.g. `Set the complete base URL for accessing the administration console, including scheme, host, port and path.`